docs/UserGuide.md

```diff
@@ -144,14 +144,21 @@ Adds module(s) to an existing contact
 Format: `addmodule INDEX m/MODULE [m/MODULE]...`
 
 * Adds modules represented by each module code `m/MODULE` to a contact at index `INDEX`
-* The index refers to the index number shown in the displayed contact list.
-* The index **must be a positive integer** 1, 2, 3, …​
+* The `Index` refers to the index number shown in the displayed contact list.
```

Comment:
I think should capitalise `Index`, more consistent with what we have above since we are formatting it.

Reply:
Good point. Updated!